tests/suite/cli_paths.rs

```diff
@@ -362,22 +362,19 @@ mod windows {
     fn install_uninstall_affect_path() {
         clitools::test(Scenario::Empty, &|config| {
             with_saved_path(&mut || {
-                let path = format!("{:?}", config.cargodir.join("bin").to_string_lossy());
+                let cfg_path = config.cargodir.join("bin").display().to_string();
+                let get_path_ = || get_path().unwrap().unwrap().to_string();
```

Comment:
This fix looks good to me, thanks for iterating!

This would have been easier to review if you had left the refactoring in a separate commit from the fix related to the winreg update (I would also put the fix for the winreg update in the same commit as the winreg update).

Reply:
The commit history has been rewritten, as promised.